src/main/java/uk/gov/hmcts/cmc/claimstore/models/offers/PartyStatement.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.cmc.claimstore.models.offers;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.Optional;
+
+@JsonInclude(JsonInclude.Include.NON_ABSENT)
+public class PartyStatement {
+
+    private final StatementType type;
+    private final MadeBy madeBy;
+    private final Offer offer;
+
+    public PartyStatement(StatementType type, MadeBy madeBy, Offer offer) {
+        this.type = type;
+        this.madeBy = madeBy;
+        this.offer = offer;
+    }
```

Comment:
is it legal to create a statment of type other than `OFFER` with non-null Offer?

Reply:
Should be illegal, but that check will be implemented with the stories that provide the feature to accept or reject an offer.